File: references/admin_templates/template_1/MVC/MVC/Controllers/CryptoCurrencyController.cs
using Microsoft.AspNetCore.Mvc;

namespace WowDash.Controllers
{
    public class CryptoCurrencyController : Controller
    {
        public IActionResult Marketplace()
        {
            return View();
        }
        public IActionResult MarketplaceDetails()
        {
            return View();
        }
        public IActionResult Portfolio()
        {
            return View();
        }
        public IActionResult Wallet()
        {
            return View();
        }
    }
}

File: references/admin_templates/template_1/CakePHP/CakePHP/src/Controller/CryptoCurrencyController.php
<?php
declare(strict_types=1);

namespace App\Controller;

/**
 * CryptoCurrency Controller
 *
 */
class CryptoCurrencyController extends AppController
{

    public function marketplace()
    {
        $this->set('title', 'Market Place');
        $this->set('subTitle', 'Market Place');
    }

    public function marketplaceDetails()
    {
        $this->set('title', 'Market Place Details');
        $this->set('subTitle', 'Market Place Details');
    }

    public function portfolio()
    {
        $this->set('title', 'Portfolio');
        $this->set('subTitle', 'Portfolio');
    }

    public function wallet()
    {
        $this->set('title', 'Wallet');
        $this->set('subTitle', 'Wallet');
    }
}
